core/src/test/java/com/linecorp/armeria/server/AnnotatedHttpServiceTest.java

```diff
@@ -558,19 +558,19 @@ public String headerDefault(RequestContext ctx,
 
         @Get("/headerWithParam")
         public String headerWithParam(RequestContext ctx,
-                                    @Header("username") @Default("hello") String username,
-                                    @Header("password") @Default("world") Optional<String> password,
-                                    @Param("extra") Optional<String> extra,
-                                    @Param("number") int number) {
+                                      @Header("username") @Default("hello") String username,
+                                      @Header("password") @Default("world") Optional<String> password,
+                                      @Param("extra") Optional<String> extra,
+                                      @Param("number") int number) {
             validateContext(ctx);
             return username + "/" + password.get() + "/" + extra.orElse("(null)") + "/" + number;
         }
 
         @Get
         @Path("/headerWithoutValue")
         public String headerWithoutValue(RequestContext ctx,
-                                    @Param("username") @Default("hello") String username,
-                                    @Param("password") String password) {
+                                         @Param("username") @Default("hello") String username,
+                                         @Param("password") String password) {
```

Comment:
Thank you~! could you replace above two `@Param` with `@Header` as well? That was my first intention. :-)